### ISSUE TITLE
Who created http://wwwb.library.vanderbilt.edu/exist/apps/srophe/bibl/820

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#487]
Hi Nathan,

I am confused about the origin of this bibl record, it looks like Winona created it in January? Do you recall anything about it?

It is not in Zotero.

D

**Comment by davidamichelson** [comment]
Note, I just edited it but in the pubplace it had "Londres" making me think it came from Zanetti somehow?

**Comment by nathangibson** [comment]
Yes, it was one of the ones I mass created from Zanetti, before we were using Zotero to export these. It is cited in http://wwwb.library.vanderbilt.edu/exist/apps/srophe/work/792 <http://wwwb.library.vanderbilt.edu/exist/apps/srophe/work/792>.

**Comment by davidamichelson** [comment]
Okay, I wonder is there a way we could batch id these, perhaps by creation
date?

**Comment by nathangibson** [comment]
Yes, you can use //change[@when='2016-01-04+01:00’] or the URI range specified as a block in the bibliography uris spreadsheet.

**Comment by nathangibson** [comment]
@davidamichelson Was there something more you wanted me to do with this?
